src/Operation/Explain.php

```diff
@@ -159,13 +159,4 @@ private function createOptions(): array
 
         return $options;
     }
-
-    private function isFindAndModify(Explainable $explainable): bool
```

Comment:
This was originally introduced in https://github.com/mongodb/mongo-php-library/commit/c6a2df9a3f4c39a2c40181a83fcfbaca66127148 and I missed removing it in https://github.com/mongodb/mongo-php-library/commit/103c1f2ea5c3099a7fccd5b22eabf33fa5eae5a7#diff-fc0bd4b2f7c5d5615660cecd2acc184702ebfbd7153723cc4d0954bc6696bc08L121 when cleaning up code for pre-3.6 servers.